cas-server-core-api-ticket/src/main/java/org/jasig/cas/ticket/ExpirationPolicy.java

```diff
@@ -24,4 +24,20 @@
      * @return true if the ticket is expired, false otherwise.
      */
     boolean isExpired(TicketState ticketState);
+
+    /**
+     * Describes the time duration where this policy should consider the item alive.
+     * Once this time passes, the item is considered expired and dead.
+     * @return alive time in seconds. A zero value indicates the time duration
+     * is not supported or is inactive.
+     */
+    Long getTimeToLive();
```

Comment:
any specific reason you decided to use the boxed object `Long` instead of the primitive `long`? can this return null?